srcs/juloo.keyboard2/Theme.java

```diff
@@ -22,25 +28,172 @@
   private final Paint _specialKeyLabelPaint;
   private final Paint _keySubLabelPaint;
   private final Paint _specialKeySubLabelPaint;
+  
+  public final boolean drawKeyBorder;
+  public final boolean drawKeyBorderTop;
+  public final boolean drawKeyBorderBottom;
+  public final boolean drawKeyBorderLeft;
+  public final boolean drawKeyBorderRight;
```

Comment:
You can have a separate class for the border styles. It will improve the code below a lot.

Reply:
I'll implement this as soon as I get the time. I might be able to do this tonight.

Reply:
Done

Reply:
I've implemented borders in 64c7c8c. It's not as elaborate as on your PR yet: no settings, no different width. However, the drawing code is better, if you want to rebase on it.

Reply:
Great. I do not think I will have time to work on this for a long while, but it is good to see you are still working on it.